deploy/ovms/values.yaml

```diff
@@ -26,11 +30,26 @@ plugin_config: >-
   {\"CPU_THROUGHPUT_STREAMS\":\"CPU_THROUGHPUT_AUTO\"}
 
 gcp_creds_secret_name: ""
-
+# k8s secret name holding gcp credential file gcp-creds.json
 aws_access_key_id: ""
 aws_secret_access_key: ""
 aws_region: ""
 s3_compat_api_endpoint: ""
-
+azure_storage_connection_string: ""
 log_level: INFO
 service_type: "LoadBalancer"
+# alternatively set to NodePort or ClusterIP
+resources: {}
+  #  limits:
+  #    cpu: 8.0
+  #    memory: 512Mi
+  #    gpu.intel.com/i915: 1
+security_context: {}
+  #  runAsUser: 5000
+  #  runAsGroup: 5000
```

Comment:
Are commented values some kind of example or did you miss removing them after development?

Reply:
they are examples